proposed/MPID Messaging.md

```diff
@@ -256,7 +244,7 @@ The various different types for `StructuredData::data` can be enumerated as:
 #[allow(variant_size_differences)]
 enum MpidMessageWrapper {
     /// Notification that the MPID Client has just connected to the network
-    Online,
+    Online(mpid_name: NameType),
```

Comment:
the sender will be the recipient_client (which is Authority::Client, i.e. the proxy), the `mpid_name` here shall be the account_name (a human readable name?)